modules/caddyhttp/reverseproxy/streaming.go

```diff
@@ -68,7 +68,7 @@ func (h *Handler) handleUpgradeResponse(logger *zap.Logger, rw http.ResponseWrit
 	//nolint:bodyclose
 	conn, brw, hijackErr := http.NewResponseController(rw).Hijack()
 	if errors.Is(hijackErr, http.ErrNotSupported) {
-		h.logger.Sugar().Errorf("can't switch protocols using non-Hijacker ResponseWriter type %T", rw)
+		h.logger.Error("can't switch protocols using non-Hijacker ResponseWriter", zap.String("type", fmt.Sprintf("%T", rw)))
```

Comment:
Just a minor cleanup. I was looking for any `.Sugar()` cause I wanted to check if we needed to set `with_caller_skip` to anything higher than 0 by default, and I think it's only relevant for sugared loggers. This was the only case except for `metrics.go` which I don't care about.